Cython/Includes/numpy/__init__.pxd

```diff
@@ -395,7 +395,7 @@ cdef extern from "numpy/arrayobject.h":
     npy_intp PyArray_DIM(ndarray, size_t)
     npy_intp PyArray_STRIDE(ndarray, size_t)
 
-    # object PyArray_BASE(ndarray) wrong refcount semantics
+    object PyArray_BASE(ndarray) #wrong refcount semantics?
```

Comment:
The [NumPy documentation](https://docs.scipy.org/doc/numpy/reference/c-api.array.html#c.PyArray_BASE) isn't clear here, but I assume that `PyArray_BASE()` is just a macro that returns a borrowed reference? If so, then the correct return type here is `PyObject *` and not a normally refcounted `object`. You'll then have to cast it to `<object>` on use, which will turn it into an owned reference.

Reply:
yes, it's just a macro. Fixing to old logic